src/codegen/unwinding.cpp

```diff
@@ -542,16 +543,20 @@ BoxedModule* getCurrentModule() {
     return compiledFunction->clfunc->source->parent_module;
 }
 
-BoxedDict* getLocals(bool only_user_visible, bool includeClosure) {
-    for (PythonFrameIterator& frame_info : unwindPythonFrames()) {
+// TODO factor getStackLoclasIncludingUserHidden and fastLocalsToBoxedLocals
+// because they are pretty ugly but have a pretty repetitive pattern.
+
+// TODO no reason for this to be a BoxedDict over an ordinary dict, I think
```

Comment:
Using a BoxedDict gives us GC support; we could probably also use a `std::unordered_map<STLCompatAllocator>` which I guess would let us not have to box the strings going in and out.